### PR TITLE
Ft bus stop markers

### PR DESCRIPTION
**Pull request by Rwandarushya**
What does this PR do?
-updated react-leaflet to the latest version
-Display all bus-stops data on the leaflet map from geojson data

Description of Task to be completed?
-create dummy geogson data of the bus stops
-display each bus stop on the map using a marker
-add a popup to give bus stop information when the marker is clicked

How should this be manually tested?
Clone the GitHub repository
-run npm install
run npm run dev to start the dev server
Any background context you want to provide?
N/A

What are the relevant pivotal tracker stories?
https://trello.com/c/wphqgRad

Screenshots (if appropriate)

Screen Shot 2020-11-22 at 17 01 07

Questions:
N/Q

### CODE REVIEW

File: src/pages/LandingPage/MapComponent.js [hunk 1]
```diff
@@ -1,77 +1,71 @@
...
-import L from 'leaflet';
-import PropTypes from 'prop-types';
-import originIcon from 'App/assets/images/origin.svg';
-import destinationIcon from 'App/assets/images/destination.svg';
+import React, { useEffect, useState } from "react";
+import { MapContainer, TileLayer } from "react-leaflet";
+import L, { Icon } from "leaflet";
+import PropTypes from "prop-types";
+// import originIcon from "App/assets/images/origin.svg";
+// import destinationIcon from "App/assets/images/destination.svg";
```
Comment (maximeish) [inline src/pages/LandingPage/MapComponent.js]: Good work @Rwandarushya, the bus stops are being correctly displayed on the map with the new react-leaflet version. Small fixes to make would be to remove the commented out code if it is not needed and also check the newline eslint error in `src/pages/LandingPage/busStopData.json`

File: src/pages/LandingPage/MapComponent.js [hunk 2]
```diff
@@ -1,77 +1,71 @@
...
-      iconSize:     [38, 95],
-      shadowSize:   [50, 64],
-      iconAnchor:   [22, 94],
-      shadowAnchor: [4, 62],
-      popupAnchor:  [-3, -76]
+const MapComponent = () => {
+  function onEachFeature(feature, layer) {
+    layer.setIcon(stopIcon);
+    layer.bindPopup(`<p>Name: ${feature.properties.name}</p>`);
+  }
```
Comment (Monehin) [inline src/pages/LandingPage/MapComponent.js]: Change this to an arrow function

File: src/pages/LandingPage/MapComponent.js (hunk 1 above)
Comment (Rwandarushya) [reply src/pages/LandingPage/MapComponent.js]: Thanks for the feedback, i resolved the above issues

File: src/pages/LandingPage/MapComponent.js (hunk 2 above)
Comment (Rwandarushya) [reply src/pages/LandingPage/MapComponent.js]: I changed the above function to arrow function

**Review by maximeish — APPROVED** [review]

**Review by IRANKUND — APPROVED** [review]
Well done @Rwandarushya

**Pull request merged**